package.json

```diff
@@ -1,22 +1,21 @@
 {
   "name": "website-welcome",
   "version": "1.0.0",
-  "description": "https://welcome.realdevsquad.com/",
-  "main": "index.js",
+  "description": "Welcome website of RealDevSquad",
```

Comment:
Where is this file coming from? 🤔 
This is a simple HTML, CSS, JS repo. Node is not required here.
Please delete this one

Reply:
Yeah even I'm not sure of this one... It was already present there, so just removed the entry point.